src/javascripts/ng-admin/Crud/misc/RestWrapper.js

```diff
@@ -64,6 +64,12 @@ define(function () {
             .customDELETE();
     };
 
+    RestWrapper.prototype.deleteAll = function(entityName, url, params) {
```

Comment:
`deleteAll` is called by `WriteQueries.js` in admin-config (see [#PR 22](https://github.com/marmelab/admin-config/pull/22/files#diff-3f65bd7f44ea89310dae2ce910e8db50R71))